SML/SML/Perceptron/Perceptron.cs

```diff
@@ -0,0 +1,248 @@
+﻿using SML.Matrices;
+
+namespace SML.Perceptrons;
+
+public class Perceptron
+{
+    #region Fields
+
+    public double[,] Input { get; set; }
+    public int Iterations { get; set; } = 10000;
```

Comment:
If it is implementation of the model in NN, it should name **epochs**
```suggestion
    public int Epochs { get; set; } = 10000;
```

Reply:
Agree